lib/rules/no-shadow.js

```diff
@@ -44,7 +44,8 @@ module.exports = {
         ],
 
         messages: {
-            noShadow: "'{{name}}' is already declared in the upper scope."
+            noShadow: "'{{name}}' is already declared in the upper scope {{shadowedLine}}:{{shadowedColumn}}.",
+            noShadowGlobal: "{{name}} is already a global variable."
```

Comment:
Quoting the name for consistency with the other message:

```suggestion
            noShadowGlobal: "'{{name}}' is already a global variable."
```

Reply:
I fixed the code following the advice.